otel/README.md

```diff
@@ -0,0 +1,35 @@
+# Tracing Demo Setup
+
+The directory contains sample [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector)
+and [Jaeger](https://www.jaegertracing.io) configurations for a tracing demo.
+
+## Configuration
+
+The provided [docker-compose.yaml](docker-compose.yaml) sets up 2 Containers
+
+1. OpenTelemetry Collector listening on port 4317 for GRPC
+2. Jaeger all-in-one listening on multiple ports
+
+The collector forwards all received spans to Jaeger over port 14250 and Jaeger exposes a UI over port `16686`.
+
+## Usage
+
+1. Start all the Containers
+```shell
+make run
+```
+
+2. Export
+
+```shell
+export OTEL_EXPORTER_OTLP_TRACES_ENDPOINT=http://localhost:4317
```

Comment:
This might need to be `OTEL_EXPORTER_OTLP_ENDPOINT` or have `/v1/traces` appended.

https://github.com/open-telemetry/opentelemetry-specification/blob/v1.8.0/specification/protocol/exporter.md#endpoint-urls-for-otlphttp

Reply:
The code is using the GRPC based endpoint so it does not need that appended